frontend/device/devicelistener.lua

```diff
@@ -310,18 +310,60 @@ function DeviceListener:onToggleFlashOnPagesWithImages()
     G_reader_settings:flipNilOrTrue("refresh_on_pages_with_images")
 end
 
-function DeviceListener:onSwapPageTurnButtons(show_notification)
-    G_reader_settings:flipNilOrFalse("input_invert_page_turn_keys")
-    Device:invertButtons()
-    if show_notification then
-        local new_text
+function DeviceListener:onSwapPageTurnButtons(side)
+    local new_text
+    if side == "left" then
+        -- revert any prior global inversions first. No shenanigans please, as we could end up with an all greyed out menu.
```

Comment:
If there are ending periods, it should start like a sentence with an uppercase.
Also don't know what is conveyed by the wording "No shenanigans please", who are you addressing ? Reads a little odd.

Reply:
I am not sure how to explain it, it's like those signs you find in parks that read "No ball games" or "No dog fouling", they are aimed generally at the public, so not odd really

Reply:
~~Well, you should try to write it in plain english, for non-english future coders to not wonder the meaning.
For now, I don't know how to read it - and these are some alternative wording that I would know how to read, pick the one that is what you meant :)~~ {I stopped writting)
You edited it, thanks for that.